pkg/snapshot/restorer/restorer_test.go

```diff
@@ -645,6 +662,222 @@ var _ = Describe("Running Restorer", func() {
 
 	})
 
+	Describe("NEGATIVE:For scenarios involving both old as well as updated directory structures being present", func() {
```

Comment:
Why did you mark it as `NEGATIVE` ? It should be like other test cases.

Reply:
Removed the `NEGATIVE` tag